plugins/realtek-mst/meson.build

```diff
@@ -0,0 +1,33 @@
+if get_option('gudev') and get_option('plugin_realtek_mst')
```

Comment:
The design pattern that is preferred is like this 
```
if get_option('plugin_realtek_mst')
if not get_option('gudev')
  error('gudev is required for plugin_realtek_mst')
endif
```

That way it's obvious if you're trying to enable realtek mst plugin but didn't enable gudev what you need to do.

Reply:
Mea culpa, fixed.